egs/wsj/s5/utils/data/get_reco2num_frames.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
```

Comment:
Is this really still needed?  I thought we discussed a different way to solve whatever issue you were trying to solve with this.